portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+
+/** Class containing comment data. */
+public final class Comment {
```

Comment:
This is a classic Java value class. Thus it's better to use @AutoValue here.
https://mincong.io/2018/08/21/why-you-should-use-auto-value-in-java/

Note this is beyond the scope of your project. But it's good to learn about it.

Reply:
Great tip, thanks a lot! Will do.